genapi.py

```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
```

Comment:
I guess I assumed that the .js and .json files should be together, but that was probably silly to assume! I'll update it to be created in the build directory.